sql-statements/sql-statement-create-table.md

```diff
@@ -75,7 +75,7 @@ KeyPart ::=
 
 IndexOption ::=
     'COMMENT' String
-|   ( 'VISIBLE' | 'INVISIBLE' )
+|   ('USING' | 'TYPE') ('BTREE' | 'RTREE' | 'HASH')
```

Comment:
Different from the English version. Please check.
![image](https://github.com/pingcap/docs-cn/assets/30922556/0b3341b5-68f2-4e67-994f-cc56f1703325)

Reply:
fixed